compat/src/forwardRef.js

```diff
@@ -33,7 +33,7 @@ export function forwardRef(fn) {
 		ref = props.ref || ref;
 		return fn(
 			clone,
-			typeof ref === 'object' && !('current' in ref) ? null : ref
+			!ref || (typeof ref === 'object' && !('current' in ref)) ? null : ref
```

Comment:
The test passes even without the change

Reply:
Yes saw that, trying to figure out why the second argument is defaulting to an empty object